python/requests/best-practice/use-request-json-shortcut.yaml

```diff
@@ -0,0 +1,16 @@
+rules:
+- id: python.requests.best-practice.use-request-json-shortcut
+  patterns:
+  - pattern-inside: import json; ...
+  - pattern-inside: import requests; ...
+  - pattern: requests.$METHOD(..., body=json.dumps($BODY), ...)
+  message: |
+    The requests library has a convenient shortcut for sending JSON requests,
+    which lets you stop worrying about serializing the body yourself.
+    To use it, replace `body=json.dumps(...)` with `json=...`.
```

Comment:
@underyx Since it's a direct, same line fix, mind adding a fix, something like:

```yaml
fix: requests.$METHOD(..., json=$BODY)
```

Reply:
Oh actually, we'd also have to grab `timeout` or other args :/

Reply:
yeah, that's exactly why, I tried a billion things (even a regex replacement) and nothing worked propery 😭